src/kibana/components/vislib/visualizations/line_chart.js

```diff
@@ -332,15 +320,17 @@ define(function (require) {
           self.addCircleEvents(circles, svg);
           self.createEndZones(svg);
 
-          var line = svg
-          .append('line')
-          .attr('class', 'base-line')
-          .attr('x1', startLineX)
-          .attr('y1', yScale(0))
-          .attr('x2', width)
-          .attr('y2', yScale(0))
-          .style('stroke', '#ddd')
-          .style('stroke-width', lineStrokeWidth);
+          var scale = (scaleType === 'log') ? yScale(1) : yScale(0);
```

Comment:
Yeah, I considered that too. Yes, it will draw the line, and yes it will draw it outside the scope of the vis, but this also seems to be the case in all the other point series vis' as well. That is, it's going to draw the axis line either way (yScale(0) is always called, since yMin is no longer passed or checked). 

I'm planning to open a new issue about drawing lines on the 0/1 axis as well as the bottom of the chart, which currently none of the visualizations do.